extensions/amp-analytics/0.1/vendors.js

```diff
@@ -458,6 +459,16 @@ export const ANALYTICS_CONFIG = {
     },
   },
 
+  'adobeanalytics_iframePing': {
+    'triggers': {
+      'pageLoad': {
+        'on': 'visible',
+        'request': 'iframeMessage',
+        'iframePing': true,
```

Comment:
I'd like for our customers to be able to distinguish why they might choose one template over another. That's how we came up with both labels. IFrame because it replaces our published iFrame solution and nativeConfig as it uses the native Adobe Analytics configuration.

My team has approved both these names, but, if there is a descriptive string that you like better I've happy to bring it back to the team.